### DISCUSSION TITLE
Humidity changing from day to day

### DISCUSSION THREAD

**Discussion by Simonthorsted** [arendst/Tasmota#11642] [Q&A] [answered]
Hello.
Can some one help me whit a rule for Tasmota  humidity ,be curse off the whether the humidity changing from day to day 
so i tought about this and was wandering if it is possible to remember the value every 30min and then compare if humidity 
raises 10 or 20 % within a  few miniuts then power 1 on for 20min and the off again, will this be possible ?

i`m not a coder so therefor i`m asking , hope someone can help, or tell me were to find a rule like that !

**Comment by barbudor** [comment, marked as answer]
There is a similar example in the Rule Cookbook here : https://tasmota.github.io/docs/Rules/#transmit-sensor-value-only-when-a-delta-is-reached
It's about temperature and MQTT publish but you can adapt to your humidity sensor and powering  instead of publication

**Reply by Simonthorsted** [reply]
Is it between polling data time the %  or were is time ?

and this i only a mqtt publish this does not do any thing to relay maybe i forgot to say it is a sonoff TH10 i`m using :-=
sorry 
Rule1
  ON SI7021#humidity>%var1% DO Backlog var1 %value%; publish stat/mqttTopic/Humidity%value%; var2 %value%; add1 2; sub2 2 ENDON
  ON SI7021#humidity<%var2% DO Backlog var2 %value%; publish stat/mqttTopic/Humidity%value%; var1 %value%; add1 2; sub2 2 ENDON

**Reply by sfromis** [reply]
Those triggers will poll "as fast as possible", every 1-2 seconds. You can avoid repeated triggers by setting the one-shot ("once") option for the rule set. Poll interval can be reduced to TelePeriod (default 5 minutes) using the tele- prefix on the triggers.

You can include other commands like Power (or whatever) in the sequence of commands in Backlog.
